java-manta-client-unshaded/src/main/java/com/joyent/manta/client/multipart/ServerSideMultipartManager.java

```diff
@@ -264,7 +252,7 @@ public ServerSideMultipartUpload initiateUpload(final String path,
 
         final int expectedStatusCode = HttpStatus.SC_CREATED;
 
-        try (CloseableHttpResponse response = connectionContext.getHttpClient().execute(post)) {
+        try (CloseableHttpResponse response = httpHelper.getConnectionContext().getHttpClient().execute(post)) {
```

Comment:
[Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) blah blah...

We may want to break this (context or client) out to a separate variable to make it easier to read.